src/utils/prompt-templates.ts

```diff
@@ -1,13 +1,116 @@
+/* eslint-disable no-console */
 import { GalileoApiClient } from '../api-client';
 import { Message, MessageRole } from '../types/message.types';
 import {
   PromptTemplate,
   PromptTemplateVersion
 } from '../types/prompt-template.types';
 
+export const getPrompts = async ({
+  name,
+  limit = 100
+}: {
+  name: string;
+  limit?: number;
+}): Promise<PromptTemplate[]> => {
```

Comment:
nit optimization: Could create a shared client instead of creating one during every call
```suggestion
let sharedClient: GalileoApiClient | null = null;

const getSharedClient = async (): Promise<GalileoApiClient> => {
  if (!sharedClient) {
    sharedClient = new GalileoApiClient();
    await sharedClient.init({ projectScoped: false });
  }
  return sharedClient;
};
const apiClient = await getSharedClient();
```